vscode/src/registry.json

```diff
@@ -74,6 +74,17 @@
           "path": "library/qtest"
         }
       }
+    },
+    {
+      "name": "QuantumArithmetic",
+      "description": "Quantum arithmetic algorithms",
+      "dependency": {
+        "github": {
+          "owner": "fedimser",
+          "repo": "quant-arith-re",
+          "path": ""
```

Comment:
If the Q# project is in the root of the repo (which yours is), you don't need to specify the path property, so you can just delete this.

Reply:
I moved the library so qsharp.josn is now in this folder: https://github.com/fedimser/quant-arith-re/tree/main/lib . So now,  as I understand, path is "lib".